pkg/manager/manager_tasks.go

```diff
@@ -165,8 +165,11 @@ func (s *singleton) Generate(sprites bool, previews bool, markers bool, transcod
 			return
 		}
 		totalsNeeded := s.neededGenerate(scenes, sprites, previews, markers, transcodes)
-		logger.Infof("Generating %d sprites %d previews %d markers %d transcodes", totalsNeeded.sprites, totalsNeeded.previews, totalsNeeded.markers, totalsNeeded.transcodes)
-
+		if totalsNeeded == nil {
```

Comment:
I think we should add an info log here just saying that it is determining job length or something, because currently there's no logging for 90 seconds, which may lead the user to thinking something is wrong.